bin/slack-alert

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+ENVIRONMENT_NAME=$1
+BUILD_NUMBER=$2
+RELEASE_ID=$3
+WEBHOOK_URL=$4
+
+RELEASE_URL="https://dev.azure.com/dfe-ssp/S118-Teacher-Payments-Service/_releaseProgress?_a=release-pipeline-progress&releaseId=$RELEASE_ID"
+
+MESSAGE=":ship: Build $BUILD_NUMBER is on its way to $ENVIRONMENT_NAME! $RELEASE_URL :rocket:"
```

Comment:
It's not on its way. At this point it should be there and restarting.

Reply:
Yeah, I said that to set the expectation that it might not _quite_ be there, but it sounds better if I actually say it's deployed